src/main/java/ch/njol/skript/util/BlockUtils.java

```diff
@@ -105,8 +107,18 @@ public static BlockData createBlockData(String dataString) {
 		
 		try {
 			return Bukkit.createBlockData(data.startsWith("minecraft:") ? data : "minecraft:" + data);
-		} catch (IllegalArgumentException ignore) {
-			return null;
+		} catch (IllegalArgumentException ignored) {
+			try {
+				// we use the original dataString param here as we want the alias before modifications
+				String alias = dataString.substring(0, data.lastIndexOf("["));
```

Comment:
_Space for improvement_
`data.lastIndexOf("[")` can be cached, not much of impact but why not if it oesn't hurt

Reply:
okay so, I originally changed this to be cached but then I realized it actually can't be because `dataString` and `data` might be different, so the last index could be different too. Thanks for the suggestion though as I wouldn't have caught this bug otherwise!

Reply:
Glad to hear that 😀 I first saw them both using `data` not `data` and `dataString` that's why I suggested, all good now.